### ISSUE TITLE
ContactShadows doesn't work

### ISSUE THREAD

**Issue by romarybintsev** [pmndrs/drei#206]
Unable to get ContactShadow to work in any way. Even the official documentation seems to not be working ([https://drei.react-spring.io/?path=/story/shaders-contactshadows--contact-shadow-st](https://drei.react-spring.io/?path=/story/shaders-contactshadows--contact-shadow-st)). Would love to get some help, thanks

**Comment by drcmda** [comment]
they work, but the must face the direction of the model and they're quite specific when it comes to how far their camera reaches: https://codesandbox.io/s/floating-shoe-mz11v

**Comment by romarybintsev** [comment]
Thanks for the provided sandbox. I'll try to reproduce the results on my end. I still think that the official example should be made clearer though since it doesn't look like anything is happening at all.

**Comment by drcmda** [comment]
if you find a better way to describe it after you have adapted it to your usecase we'd be glad if you could put that into a pr. the thing is hard to control, it's kind of the point: the far prop controls how far it can "see", so that shadows disappear when the target is not in proximity. that usually takes some fiddling to get right.

**Comment by romarybintsev** [comment]
I got it working, thanks for the help. On a separate note, I'm working with sprites and noticed that they were hidden behind the "transparent" plane of <ContactShadows />. To fix it I had to add `depthTest: false`  to the [meshBasicMaterial](https://github.com/pmndrs/drei/blob/0f9fd9e05b23c348bf72af7c4698fe8ce00bb100/src/ContactShadows.tsx#L84). I thought maybe it would be good to either add it or at least have it as a variable parameter.
